website_membership_random_order/templates/website_membership_templates.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="index" inherit_id="website_membership.index">
+        <xpath expr="//t[@t-if='not memberships_partner_ids']" position="after">
+            <t t-else="">
+                <p
+                    style="text-align: center"
```

Comment:
Could this be styled a little differently to help differentiate it from the rest of the page content?

Reply:
Done:

![Selección_018](https://user-images.githubusercontent.com/7165771/133564383-c1c08ccc-923c-486e-a661-da02fb34ee74.png)

Reply:
Nice! Much better IMHO :smiley: